Zephyr modules failure report

During CMake configure time the script `zephyr_module.py` is called to find potential Zephyr modules.

`zephyr_module.py` uses `west list`, however if `west list` fails, the error from `west` is swallowed by `zephyr_module.py` and instead a new error is raised making it harder to find the root cause.

Existing error raised looks as:
```
CMake Error at C:/projects/github/ncs/zephyr/cmake/zephyr_module.cmake:40 (message):
  Traceback (most recent call last):

    File "C:/projects/github/ncs/zephyr/scripts/zephyr_module.py", line 165, in <module>
      main()
    File "C:/projects/github/ncs/zephyr/scripts/zephyr_module.py", line 139, in main
      returncode=p.returncode)

  subprocess.CalledProcessError: Command '['west', 'list',  '--format={posixpath}']' returned non-zero exit status 1.

Call Stack (most recent call first):
  C:/projects/github/ncs/zephyr/cmake/app/boilerplate.cmake:246 (include)
  CMakeLists.txt:11 (include)

-- Configuring incomplete, errors occurred!
```

--- a/scripts/zephyr_module.py
+++ b/scripts/zephyr_module.py
@@ -134,6 +134,7 @@ def main():
             # outside a west managed project.
             projects = []
         else:
+            print(err.decode(sys.getdefaultencoding()))
             # A real error occurred, raise an exception
             raise subprocess.CalledProcessError(cmd=p.args,
                                                 returncode=p.returncode)

